### PR TITLE
Dreamsicle

### PR DESCRIPTION
**Pull request by willisart**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,18 @@
...
 [![Gitter chat](https://badges.gitter.im/nightscout.png)](https://gitter.im/nightscout/public)
-[![Stories in  Ready](https://badge.waffle.io/nightscout/cgm-remote-monitor.png?label=ready&title=Ready)](https://waffle.io/nightscout/cgm-remote-monitor)
-[![Stories in Progress](https://badge.waffle.io/nightscout/cgm-remote-monitor.png?label=in+progress&title=In+Progress)](https://waffle.io/nightscout/cgm-remote-monitor)
 
 [![Deploy to Heroku](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
 
-This acts as a web-based CGM (Continuous Glucose Monitor) to allow
-multiple caregivers to remotely view a patient's glucose data in
-real time.  The server reads a MongoDB which is intended to be data
+This acts as a web-based CGM (Continuous Glucose Montinor) to allow
```
Comment (sulkaharo) [inline README.md]: Line changes correct spelling to a typo

File: COPYRIGHT [hunk 2]
```diff
@@ -1 +1,5 @@
-Copyright (C) 2015 The Nightscout Foundation, http://www.nightscoutfoundation.org
```
Comment (sulkaharo) [inline COPYRIGHT]: Why remove copyright notice?

Comment (willisart) [inline COPYRIGHT]: I have no idea.  I just clicked the green button.   

Patricia Willis
www.plwillis.com

File: README.md [hunk 3]
```diff
@@ -46,96 +61,40 @@ ready, just host your web app on your service of choice.
...
+GUI.
 
 More questions?
 ---------------
 
 Feel free to [post an issue][issues], but read the [wiki][wiki] first.
 
-[issues]: https://github.com/nightscout/cgm-remote-monitor/issues
-[wiki]: https://github.com/nightscout/cgm-remote-monitor/wiki
+[issues]: https://github.com/rnpenguin/cgm-remote-monitor/issues
```
Comment (sulkaharo) [inline README.md]: Fix points to old repository, the nightscout one is the correct repository to point to.

File: README.md [hunk 4]
```diff
@@ -46,96 +61,40 @@ ready, just host your web app on your service of choice.
...
+[wiki]: https://github.com/rnpenguin/cgm-remote-monitor/wiki
```
Comment (sulkaharo) [inline README.md]: Fix points to old repository, the nightscout one is the correct repository to point to.

File: bower.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nightscout",
-  "version": "0.6.0",
+  "version": "0.5.1",
```
Comment (sulkaharo) [inline bower.json]: Merge glitch?

File: COPYRIGHT (hunk 2 above)
Comment (sulkaharo) [inline COPYRIGHT]: LOL right - you've accidentally created a pull request for the developers. Please disregard the rest of the comments! Apologies for comment spam. :D

Comment (willisart) [inline COPYRIGHT]: Oh!   Sorry about that!  I was really hoping you weren't going to ask me to fix the pull request.  Lol. Clearly, I have no idea what I am doing.   

Patricia Willis
www.plwillis.com

**Pull request closed without merging**